src/hooks/useIsKeyboardVisible.ts

```diff
@@ -3,7 +3,7 @@ import { Keyboard } from 'react-native'
 
 // This hook listens to the keyboard and returns boolean whether keyboard is opened or closed
 
```

Comment:
Optional: I think this comment is kind of redunant/describes implementation details and could therefore be removed.